charts/ovirt-csi-driver-4.20.0/templates/csi_driver_controller.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: csi-ovirt-controller
+  name: {{ .Values.csiController.ovirtController.name }}
+  namespace: '{{ .Release.Namespace }}'
```

Comment:
The single quotes around all the `namespace` definitions can be removed.

Reply:
fixed